frontend/src/metabase/visualizations/echarts/cartesian/model/transforms.ts

```diff
@@ -0,0 +1,45 @@
+import type { NumericScale, StackType } from "metabase-types/api";
+import { isNumber } from "metabase/lib/types";
+import type { NumericAxisScaleTransforms } from "./types";
+
+function getSign(value: number) {
+  return value >= 0 ? 1 : -1;
+}
+
+function signedSquareRoot(value: number) {
+  return getSign(value) * Math.sqrt(Math.abs(value));
+}
+
+export function getYAxisScaleTransforms(
+  scale: NumericScale | undefined,
+  stackType?: StackType,
+): NumericAxisScaleTransforms {
+  if (scale === "pow") {
+    return {
+      toEChartsAxisValue: value => {
+        if (!isNumber(value)) {
+          return null;
+        }
+        // Transformation for stacked charts occurs in model/dataset.ts
+        if (stackType != null) {
```

Comment:
I don't think we should add it here. These transforms correctly transform `y` values. And for both `toEChartsAxisValue` and `fromEChartsAxisValue` it would be incorrect to use them to get `y2-y1` values for stacks

Reply:
We do need this check, without it the tick labels and data labels are incorrect.

![Screenshot 2024-03-21 at 11.03.28 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/019dae3e-aa3a-472b-ada8-40287c1a814e.png)

This screenshot is with the 
```typescript
        if (stackType != null) {
          return value;
        }
```
check. As you can see the tick labels and data labels are correct.

![Screenshot 2024-03-21 at 11.05.12 AM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/9ULpbAzGBtvQRyZEpGvu/8a68102b-6093-4d70-a99c-fca55a27ffaf.png)

This is what it looks like if I remove the check, the tick and data labels are incorrect now.